### ISSUE TITLE
Showing minus value

### ISSUE THREAD

**Issue by Robert-Ordis** [lcallarec/live-chart#35]
https://twitter.com/Robert_Ordis/status/1504480343432507393

This is my app showing the timeline of the controller's input.(based on RetroSpy)
Upper chart shows this analog stick's inputs on raw value.
Lower chart shows the angle of this analog's input.
So, the "angle" shows the value from -180 to 180.
But, this chart doesn't show the "minus value".

**Comment by lcallarec** [comment]
Yes, this is a know limitation. I know that it will require some amount of work, but I'll take a look.

**Comment by OlegAushev** [comment]
Hi! Are there any plans to implement the displaying of negative values? Or maybe there is already a workaround?

**Comment by Robert-Ordis** [comment]
> Hi! Are there any plans to implement the displaying of negative values? Or maybe there is already a workaround?

I've already implemented it in my fork.（And Im waiting the author to merge my pull requests including the mod for nega val. ）
In my fork, I'm still not ready for test code.
